### DISCUSSION TITLE
BUG: Legacy TSML Table is Missing Location Data When Filtering

### DISCUSSION THREAD

**Discussion by Grateful-Viv** [code4recovery/12-step-meeting-list#835] [Q&A] [answered]
Switching to any particular day, region, type, etc. is missing the data for location looking at the meeting list table. Though the column headers are there, the only cells displayed is Time, Meeting, Address & Region. Location data is not present.

If I select a meeting and then select Back to Meetings, it shows the table correctly, until I try to filter something else.

I realize I have CSS for the legacy plugin, but I took it out on another staging site and it's still doing this.

The plugin version is 3.14.4.

Pictures below are before and after.

![Meetings Good List-–-SDIAA-Staging-TWO](https://user-images.githubusercontent.com/62247421/174682138-528ff33d-36f3-4c0b-9baf-a7905fad88f2.png)

![Meetings Bad List-–-SDIAA-Staging-TWO](https://user-images.githubusercontent.com/62247421/174682154-944d8130-3dfc-4d48-8832-ae9a2c81f7ca.png)

You can also reproduce this behavior on the live site > https://sussexaa.org/ 

Interestingly enough, the other one staging site I have the code for only showing location data and not group if it's online is not doing this, no matter how much filtering I do and undo.

**Comment by joshreisner** [comment, marked as answer]
oops. in implementing the location-only fix for you i inadvertently messed up filtering for sites that are using the default `Location/  Group` setting

**Reply by Grateful-Viv** [reply]
Is it an easy fix? It's so weird this behavior doesn't happen on TSML UI.

**Reply by joshreisner** [reply]
should be. i've created a [pull request](https://github.com/code4recovery/12-step-meeting-list/pull/836) -- we'll need other team members to help review and release

it makes sense that it would work in TSML UI -- that's a different code base

**Reply by Grateful-Viv** [reply]
You're pushing me to roll out TSML UI :)

**Reply by joshreisner** [reply]
Update: this PR has been merged and will roll out as part of the next TSML release. I'm going to go ahead and mark this as the "answered" even though I know it's not resolved yet :)

Marking as answered will help us see which issues still need answers
